script.js

```diff
@@ -0,0 +1,234 @@
+const keysLayout = [
+  { code: 'Backquote', eng: '`', rus: 'ё'},
+  { code: 'Digit1', eng: '1', rus: '1'},
+  { code: 'Digit2', eng: '2', rus: '2'},
+  { code: 'Digit3', eng: '3', rus: '3'},
+  { code: 'Digit4', eng: '4', rus: '4'},
+  { code: 'Digit5', eng: '5', rus: '5'},
+  { code: 'Digit6', eng: '6', rus: '6'},
+  { code: 'Digit7', eng: '7', rus: '7'},
+  { code: 'Digit8', eng: '8', rus: '8'},
+  { code: 'Digit9', eng: '9', rus: '9'},
+  { code: 'Digit0', eng: '0', rus: '0'},
+  { code: 'Minus', eng: '-', rus: '-'},
+  { code: 'Equal', eng: '=', rus: '='},
+  { code: 'Backspace', eng: 'Backspace', rus: 'Backspace'},
+
+  { code: 'Tab', eng: 'Tab', rus: 'Tab'},
+  { code: 'KeyQ', eng: 'q', rus: 'й'},
+  { code: 'KeyW', eng: 'w', rus: 'ц'},
+  { code: 'KeyE', eng: 'e', rus: 'у'},
+  { code: 'KeyR', eng: 'r', rus: 'к'},
+  { code: 'KeyT', eng: 't', rus: 'е'},
+  { code: 'KeyY', eng: 'y', rus: 'н'},
+  { code: 'KeyU', eng: 'u', rus: 'г'},
+  { code: 'KeyI', eng: 'i', rus: 'ш'},
+  { code: 'KeyO', eng: 'o', rus: 'щ'},
+  { code: 'KeyP', eng: 'p', rus: 'з'},
+  { code: 'BracketLeft', eng: '[', rus: 'х'},
+  { code: 'BracketRight', eng: ']', rus: 'ъ'},
+  { code: 'Delete', eng: 'DEL', rus: 'DEL'},
+
+  { code: 'CapsLock', eng: 'Caps Lock', rus: 'Caps Lock'},
+  { code: 'KeyA', eng: 'a', rus: 'ф' },
+  { code: 'KeyS', eng: 's', rus: 'ы' },
+  { code: 'KeyD', eng: 'd', rus: 'в' },
+  { code: 'KeyF', eng: 'f', rus: 'а' },
+  { code: 'KeyG', eng: 'g', rus: 'п' },
+  { code: 'KeyH', eng: 'h', rus: 'р' },
+  { code: 'KeyJ', eng: 'j', rus: 'о' },
+  { code: 'KeyK', eng: 'k', rus: 'л' },
+  { code: 'KeyL', eng: 'l', rus: 'д' },
+  { code: 'Semicolon', eng: ';', rus: 'ж'},
+  { code: 'Quote', eng: '\'', rus: 'э'},
+  { code: 'Enter', eng: '\n', rus: '\n'},
+
+  { code: 'ShiftLeft', eng: 'Shift', rus: 'Shift'},
+  { code: 'IntlBackslash', eng: '\\', rus: '\\'},
+  { code: 'KeyZ', eng: 'z', rus: 'я'},
+  { code: 'KeyX', eng: 'x', rus: 'ч'},
+  { code: 'KeyC', eng: 'c', rus: 'с'},
+  { code: 'KeyV', eng: 'v', rus: 'м'},
+  { code: 'KeyB', eng: 'b', rus: 'и'},
+  { code: 'KeyN', eng: 'n', rus: 'т'},
+  { code: 'KeyM', eng: 'm', rus: 'ь'},
+  { code: 'Comma', eng: ',', rus: 'б'},
+  { code: 'Period', eng: '.', rus: 'ю'},
+  { code: 'ArrowUp', eng: '↑', rus: '↑'},
+  { code: 'Slash', eng: '/', rus: '.'},
+
+  
+  { code: 'ControlLeft', eng: 'Ctrl', rus: 'Ctrl'},
+  { code: 'MetaLeft', eng: 'Win', rus: 'Win'},
+  { code: 'AltLeft', eng: 'Alt', rus: 'Alt'},
+  { code: 'Space', eng: 'SPACE', rus: 'SPACE'},
+  { code: 'ArrowLeft', eng: '←', rus: '←'},
+  { code: 'ArrowDown', eng: '↓', rus: '↓'},
+  { code: 'ArrowRight', eng: '→', rus: '→'},
+]
+
+const Keyboard = {
+  elements: {
+    main: null,
+    keysContainer: null,
+    keys: [],
+    textarea: null
+  },
+
+  eventHandlers: {
+    oninput: null,
+  },
+
+  properties: {
+    value: "",
+    capsLock: false,
+    rus: false
```

Comment:
Unsupportable way of storing active language. It would be better to store `Map<KeyCode, DisplayValue>` inside `activeLanguage` property so it would be possible to have any desired amount of languages.